bigchaindb/web/views/base.py

```diff
@@ -23,7 +23,7 @@ def make_error(status_code, message=None):
 
 def base_ws_uri():
     """Base websocket uri."""
-    scheme = config['wsserver']['scheme']
-    host = config['wsserver']['host']
-    port = config['wsserver']['port']
+    scheme = config['wsserver']['advertised_scheme']
```

Comment:
Maybe you could add to the docstring of this function to say how it's supposed to be used (and not used).